app/app.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+
+@Component({
+    selector: 'weather-app',
+    templateUrl: './app.component.html',
+    styleUrls: ['./styles/app.less'],
+    encapsulation: ViewEncapsulation.None
```

Comment:
Never use it) 
I know that in our lection it was mentions as most preferrable case, but it's a mistake. 

One of the most intresting things in angular 2 - it style encapsualtion and we should use. 
I agree that we some times need styles, wich will be available around hole application. 
For examle grids, or main colors or etc. I that case I'd probably agree with using encapsulation none, but actuallly even in that case I'd preffer to include it in app as separate file.